pkgs/servers/zoneminder/default.nix

```diff
@@ -201,5 +201,8 @@ in stdenv.mkDerivation rec {
     license = licenses.gpl3;
     maintainers = [ ];
     platforms = platforms.unix;
+
+    # https://github.com/ZoneMinder/ no longer exists
```

Comment:
The url is working now?

Reply:
looks like the user was removed from github

Reply:
I can't see the issue at all
https://pikwy.com/web/61d55053506dc35bac3ea661

Reply:
... what...

It was not there yesterday....